README.md

```diff
@@ -1 +1,9 @@
-# Tic-Tac-Toe
\ No newline at end of file
+# Tic-Tac-Toe
+
+This is the third project of the Main Ruby curriculum at Microverse - @microverseinc
```

Comment:
Maybe you can eventually add the instructions of how to run the game now or in the future. E.g `ruby main.rb` or if you have dependencies you plan to use `bundle install` etc. [OPTIONAL]